camelot/core/orm/entity.py

```diff
@@ -348,7 +351,6 @@ def get_cls_by_type(cls, _type):
                 return cls._cls_for_type.get(_type)
             LOGGER.warn("No registered class found for '{0}' (of type {1})".format(_type, type(_type)))
             raise UserException("No registered class found for '{0}' (of type {1})".format(_type, type(_type)))
```

Comment:
ik denk niet dat dit een user exception moet zijn :)

de user kan hier niet veel aan veranderen

Reply:
Hmm idd, we hebben dat ooit toegevoegd denk ik omdat dat dat toen zich soms voordeed bij het aanmaken van een transactie met de mode_name leeg was ofzo. We hebben daar toen achter aan het zoeken geweest, maar niet kunnen reproduceren, en dat toegevoegd om dat er uit te laten springen. Maar we hebben daar toch niets meer van gehoord, dus vermoedelijk gaat dat onrechtstreeks opgelost geweest zijn.